helm-charts/infisical/templates/NOTES.txt

```diff
@@ -68,13 +68,17 @@
 │   $ helm uninstall {{ .Release.Namespace }} {{ .Release.Name }}
 │
 │   → Get MongoDB root password
-│   $ kubectl get secret {{ .Release.Namespace }} mongodb 
+│   $ kubectl get secret -n {{ .Release.Namespace }} mongodb
 │     -o jsonpath="{.data['mongodb-root-password']}" | base64 -d
 │
 │   → Get MongoDB users passwords
-│   $ kubectl get secret {{ .Release.Namespace }} mongodb 
+│   $ kubectl get secret -n {{ .Release.Namespace }} mongodb
 │     -o jsonpath="{.data['mongodb-passwords']}" | base64 -d 
 │
+│   → Export your backend secrets (requires jq)
+│   $ kubectl get secrets/<your-secret-name> -n {{ .Release.Namespace }} \
+│     -o json | jq '.data | map_values(@base64d)' > <dest-filename>.bak
```

Comment:
oh this is where the `.bak` file is from

Reply:
Also running

```
kubectl get secrets/infisical-backend -n dev \
│     -o json | jq '.data | map_values(@base64d)' > <dest-filename>.bak
```

didn't work for me. I got an error 
```
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
```

However, running it in a single line worked 

```
kubectl get secrets/infisical-backend -n dev -o json | jq '.data | map_values(@base64d)' > backup.bak
```

Reply:
Yeah that'd be great, lets do that